[MAINTENANCE] Book - Pong Tutorial - Bump amethyst to 0.15

## Description

“[Creating a new project](https://book.amethyst.rs/stable/pong-tutorial/pong-tutorial-01.html)” section of the book refers to version = "0.14" (twice, for Vulcan and for Metal configurations). It should be probably updated to 0.15.

## Reason

Example with 0.14 currently fails with “error[E0433]: failed to resolve: could not find `__rt` in `quote`” in err-derive 0.1.6, while 0.15 uses err-derive 0.2.3 and doesn't seem to have this issue. 

## Impact

I didn't test the full example from the book yet, just the code in “[Drawing the paddles](https://book.amethyst.rs/stable/pong-tutorial/pong-tutorial-02.html)” section, so I am not sure if it is affected.

--- a/book/src/pong-tutorial/pong-tutorial-01.md
+++ b/book/src/pong-tutorial/pong-tutorial-01.md
@@ -19,7 +19,7 @@ authors = []
 edition = "2018"
 
 [dependencies.amethyst]
-version = "0.14"
+version = "0.15"
 features = ["vulkan"]
 ```
 
@@ -27,7 +27,7 @@ Alternatively, if you are developing on macOS, you might want to use the `metal`
 
 ```toml
 [dependencies.amethyst]
-version = "0.14"
+version = "0.15"
 features = ["metal"]
 ```
 

